init.cmd

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+if [%1]==[] (set /p SOLUTION_NAME="Enter Solution name: ") else (set SOLUTION_NAME=%1)
+if [%2]==[] (set /p PROJECT_NAME="Enter Project name: ") else (set PROJECT_NAME=%2)
+if [%3]==[] (set /p OUTPUT_PATH="Enter Solution directory: ") else (set OUTPUT_PATH=%3)
```

Comment:
We should move the /Terraform folder inside of the Project Assembly to integrate with CircleCI config.

Reply:
We moved Terraform folder out of Project folder. Then, this can't be a problem anymore.